addons/context_actions/functions/fnc_toggleSurrender.sqf

```diff
@@ -0,0 +1,27 @@
+#include "script_component.hpp"
+/*
+ * Author: Eclipser
+ * Toggles ACE surrendering states of units
```

Comment:
Missing period at the end. Maybe; "Toggles the ACE surrendering state of the given units."